Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -421,6 +421,9 @@ void Adjuster::adjust(RenderStyle& style, const RenderStyle* userAgentAppearance
         style.setEffectiveDisplay(DisplayType::Block);
     }
 
+    if (style.display() == DisplayType::None && style.position() == PositionType::Absolute)
```

Comment:
I wonder if we should do this check at `getComputedStyle` time instead, so we don't pay the cost of running this code on every single style adjust.

Reply:
@smfr - Done and locally tested that we are progressing WPT still with new change. Thanks for your input and help!